packages/ember-css-modules/lib/htmlbars-plugin/utils.js

```diff
@@ -21,7 +21,7 @@ function updateStringValue(node, updater) {
     node.chars = updater(node.chars);
   } else if (node.type === 'StringLiteral') {
     node.value = updater(node.value);
-    node.original = updater(node.original);
+    setValue(node, updater(getValue(node)));
```

Comment:
We're already assigning `node.value` unconditionally on the line above this one, so I'm not sure if this is quite right.

This might need to be something like:

```suggestion
    // Avoid `original` deprecation in Ember 5.9+
    let originalDesc = Object.getOwnPropertyDescriptor(node, 'original');
    if (originalDesc && originalDesc.value) {
      node.original = updater(node.original);
    }
```

And then we can just skip `getValue` and `setValue` in this module.

Reply:
I defer to your wisdom 😃